src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -285,6 +285,31 @@ private void UpdateOrganelleUnlockTooltips()
         }
     }
 
+    private void UpdateOrganelleLAWKSettings()
+    {
+        // Don't use placeablePartSelectionElements as the thermoplast isn't placeable yet but is LAWK-dependent
+        foreach (var organelle in allPartSelectionElements.Keys)
+        {
+            var control = allPartSelectionElements[organelle];
+            control.Visible = !Editor.CurrentGame.GameWorld.WorldSettings.LAWK || organelle.LAWK;
+        }
+    }
+
+    private void UpdateDifficultyAdjustedMPCost()
```

Comment:
What's the difference to `UpdateTooltipMPCostFactors` seems like there's now two methods that basically do the same thing... I think these should be combined to reduce (my) confusion.